packages/quantic/cypress/e2e/default-2/recommendation-list/recommendation-list-expectations.ts

```diff
@@ -110,10 +117,55 @@ export function recommendationListExpectations(
         );
     },
 
-    logRecommendationOpen: () => {
-      cy.wait(InterceptAliases.UA.RecommendationOpen).logDetail(
-        'should log the recommendation open UA event'
-      );
+    logRecommendationOpen: (index: number, recommendationsAlias: string) => {
+      cy.wait(InterceptAliases.UA.RecommendationOpen)
+        .then((interception) => {
+          const analyticsBody = interception.request.body;
+          const customData = analyticsBody?.customData;
+          cy.get<Array<Result>>(recommendationsAlias).then(
+            (recommendations) => {
+              const recommendation = recommendations[index];
+              expect(analyticsBody).to.have.property(
```

Comment:
It sucks the chai doesn't support this like jest does, because I just discovered this and it's amazing haha:

```
expect(obj).toEqual(
    expect.objectContaining({
      id: '111'
    })
  );
```

Basically does a partial match on an object, `expect(obj).to.partial.equal({documentTitle: recommendation.title, documentUri: recommendation.uri, ...})`